packages/core/content-releases/server/src/controllers/release-action.ts

```diff
@@ -81,9 +81,11 @@ const releaseActionController = {
     });
     const query = await permissionsManager.sanitizeQuery(ctx.query);
 
+    query.sort = query.groupBy === 'action' ? 'type' : query.groupBy;
+    delete query.groupBy;
```

Comment:
Probably not something related to this PR, but this should contain a yup validation. I don't think the sanitizeQuery is sanitizing non existing filters like "groupBy"